src/os/source/vectorsource.js

```diff
@@ -1683,6 +1683,22 @@ os.source.Vector.prototype.getFilteredFeatures = function(opt_allTime) {
       this.timeModel.intersection(range, true);
     }
 
+    // Hide annotations if needed
+    if (this instanceof plugin.file.kml.KMLSource) {
+      for (var i = 0, n = this.getFeatures().length; i < n; i++) {
+        var feature = this.getFeatures()[i];
+        var source = /** @type {plugin.file.kml.KMLSource} */ (this);
+        var annotation = source.getFeatureNode(feature).getAnnotation();
+        if (annotation) {
+          if (features.includes(feature)) {
```

Comment:
`Array.includes` is not implemented by the minimum supported browsers for Open Sphere (currently Chrome 36 and Firefox 38). Please use `ol.array.includes` instead.